### PR TITLE
Front end

### PR DESCRIPTION
**Pull request by raptattention**

### CODE REVIEW

File: www/js/controllers.js [hunk 1]
```diff
@@ -17,59 +24,50 @@ angular.module('starter.controllers', [])
...
 })
 
-.controller('InvitationsCtrl', function($scope, $http, $window) {
-  $http.get(apiUrl+'/users/'+$scope.currentUser.id+'/received').success(function(response){
-    $scope.invitationsReceived = response;
-  }).error(function(response){
-    console.log(response);
-  })
+.controller('InvitationsCtrl', function($scope, $http, $window, $location) {
+  var getReceived = function(){
```
Comment (h8rry) [inline www/js/controllers.js:79]: Need a better naming for this function

File: www/js/controllers.js [hunk 2]
```diff
@@ -322,36 +308,59 @@ angular.module('starter.controllers', [])
...
+    //   $http.post(apiUrl+'/groups/'+response.id+'/group_users', userData).success(function(resp){
+    //     console.log(resp);
+    //     $window.location.reload(true);
+    //     $location.path('/tab/groups');
+    //   }).error(function(resp){
+    //     console.log(resp);
+    //   })
+    // }).error(function(response){
+    //   console.log(response);
+    // })
```
Comment (h8rry) [inline www/js/controllers.js]: Don't leave commented code here.

File: www/js/controllers.js [hunk 3]
```diff
@@ -322,36 +308,59 @@ angular.module('starter.controllers', [])
...
-        $window.location.reload(true);
-        $location.path('/tab/groups');
-      }).error(function(resp){
-        console.log(resp);
-      })
-    }).error(function(response){
-      console.log(response);
-    })
-  }
+    $cordovaFileTransfer.upload(apiUrl+'/groups', $scope.image.file, options)
```
Comment (h8rry) [inline www/js/controllers.js]: Can you get the image path now?

File: www/templates/tab-addgroup.html [hunk 4]
```diff
@@ -5,8 +5,11 @@
         <span class="input-label">Name</span>
         <input type="text" placeholder="Name of Group" ng-model="groupData.name">
       </label>
+      <label class="item item-input">
+        <span class="input-label">Image</span>
+        <input ngf-select ng-model="image.file" type="file" id="add_item_pic_id" name="pic" accept="image/*">
```
Comment (h8rry) [inline www/templates/tab-addgroup.html:10]: Are you using using `ngf-select`?

**Comment by h8rry** [comment]
@raptattention please review

File: www/js/controllers.js [hunk 5]
```diff
@@ -345,13 +405,44 @@ angular.module('starter.controllers', [])
...
         console.log(resp);
+        $scope.hide($ionicLoading);
       })
     }).error(function(response){
       console.log(response);
+      $scope.hide($ionicLoading);
     })
-  }
+
+    // var options = {
```
Comment (f3r) [inline www/js/controllers.js:420]: Next time please create a separate branch when you want to implement new features, so you don't have commented code like this :sob:

File: www/templates/tab-addgroup.html [hunk 6]
```diff
@@ -5,8 +5,11 @@
         <span class="input-label">Name</span>
         <input type="text" placeholder="Name of Group" ng-model="groupData.name">
       </label>
+      <!-- <label class="item item-input">
```
Comment (f3r) [inline www/templates/tab-addgroup.html:8]: and this :sob:

**Pull request merged**